### PR TITLE
clear interface on leaving network

### PR DESCRIPTION
**Pull request by alphadose**

### CODE REVIEW

File: functions/uninstall.go [hunk 1]
```diff
@@ -56,20 +56,20 @@ func LeaveNetwork(network string, isDaemon bool) ([]error, error) {
...
-		nc.Create()
-		if err := nc.Configure(); err != nil {
-			faults = append(faults, fmt.Errorf("failed to configure interface during node removal - %v", err.Error()))
-		} else {
-			if err = wireguard.SetPeers(); err != nil {
-				faults = append(faults, fmt.Errorf("issue setting peers after node removal - %v", err.Error()))
-			}
+	// re-configure interface
+	nc := wireguard.GetInterface()
+	nc.Iface.Close()
```
Comment (abhishek9686) [inline functions/uninstall.go:61]: these steps should only be called from daemon

**Review by theguy951357 — COMMENTED** [review]
will wait on changes

**Review by mattkasun — COMMENTED** [review]
approved subject to addressesing Abi's comment

**Review by mattkasun — APPROVED** [review]

**Comment by theguy951357** [comment]
is this one ready to test or is it still waiting on changes?

**Comment by abhishek9686** [comment]
these changes are not required

**Pull request closed without merging**